<!DOCTYPE html>
<html><head><title>Catch</title><link rel="stylesheet" href="../../styles.css"/><script src="../../scripts.js"></script></head><body onload="ro();">
<div class="rH">4 references to Catch</div><div class="rA">Assembly-CSharp (1)</div><div class="rG" id="Assembly-CSharp"><div class="rF"><div class="rN">Assets\UnitTests\UnitTests.cs (1)</div>
<a href="../../Assembly-CSharp/Assets/UnitTests/UnitTests.cs.html#1741"><b>1741</b>.<i>Catch</i>((InvalidOperationException ex) =&gt;</a>
</div>
</div>
<div class="rA">Assembly-CSharp-firstpass (3)</div><div class="rG" id="Assembly-CSharp-firstpass"><div class="rF"><div class="rN">Assets\Plugins\UniRx\Scripts\Observable.ErrorHandling.cs (3)</div>
<a href="../Assets/Plugins/UniRx/Scripts/Observable.ErrorHandling.cs.html#29"><b>29</b>return source.<i>Catch</i>&lt;TSource, Exception&gt;(Stubs.CatchIgnore&lt;TSource&gt;);</a>
<a href="../Assets/Plugins/UniRx/Scripts/Observable.ErrorHandling.cs.html#36"><b>36</b>var result = source.<i>Catch</i>((TException ex) =&gt;</a>
<a href="../Assets/Plugins/UniRx/Scripts/Observable.ErrorHandling.cs.html#118"><b>118</b>self = source.<i>Catch</i>((TException ex) =&gt;</a>
</div>
</div>
</body></html>